src/clients/requestFilter.ts

```diff
@@ -0,0 +1,22 @@
+import Redis from 'ioredis';
+
+const redis = new Redis(process.env.REDIS_URL);
```

Comment:
Please update the `README.md` file and add the environment variable `REDIS_URL`. Also, please add a quick guide on how to get this value.